kagabof/tic_tac_toe
# ft: project setup and logic implementation

#### Description of changes made
---

> - project setup and logic implementation

#### Type of change
---

> **Please select the relevant options**


#### How Has This Been Tested?
---

> **Please describe the tests that you ran to verify your changes**


#### How should this be manually tested?
---

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/kagabof/tic_tac_toe.git), and `cd tic_tac_toe`
- Check out this branch by `git checkout [the branch for this pr]`
- Run `npm i` to install the project dependencies
- Run `npm run dev` for development or `npm start` production env.
- .....

#### Checklist:



#### Screenshots

| Request and response |
| :------------------: |
|  ![Screen Shot 2022-03-21 at 14 34 12](https://user-images.githubusercontent.com/44500114/159261918-65c8244e-8611-4fc7-b5bb-65e49440a7a2.png)
      |
